docs/03-Metrics/plugins/readme.md

```diff
@@ -4,6 +4,8 @@ Each metric is associated with a Plugin.
 Associated metrics are linked below.
 See [Metrics Configuration](../configuration.md) for info on configuration.
 
+To run Retina without any plugins, the `CAP_BPF` Linux capability is required for memory locking and for loading/using BPF programs. This capability is mandatory. If you're using any plugins, ensure that the necessary capabilities for those plugins are also added. If a plugin requires `CAP_SYS_ADMIN`, you can substitute it for `CAP_BPF`.
```

Comment:
> If a plugin requires `CAP_SYS_ADMIN`, you can substitute it for `CAP_BPF`.
Does this actually work? Last time I tested this it didn't 🥲

Reply:
Yes @nddq! I created the retina binary and ran it with different capabilities with and without plugins

Reply:
Can you make a change to the helm deployment files to test this out in our pipeline? Also, I remember `CAP_BPF` is only available from a specific kernel version onward, so we should clarify it as well.

Reply:
Done! Updated in latest commit